test/browser.js

```diff
@@ -85,11 +85,13 @@ test('throws TimeoutError even though it does not support AbortController', with
 	await page.addScriptTag({path: './test/helpers/disable-abort-controller.js'});
 	await page.addScriptTag({path: './umd.js'});
 
+	let request;
 	const error = await page.evaluate(url => {
-		const request = window.ky(`${url}/slow`, {timeout: 500}).text();
+		request = window.ky(`${url}/slow`, {timeout: 500}).text();
```

Comment:
Oh, wow. Not being familiar with Puppeteer, I find that extremely surprising. By just looking at the code snippet in isolation it's not intuitive at all. But as always, you should know what you're running, I guess. 

Nevertheless, nice catch. Classic mistake of skipping the red part of red-green-refactor...

Anyhow, I'll revert the changes to this test. Do you want me to add a new test file for `TimeoutError` instead?

Side-note: did you see my comments in #300 regarding `node-fetch`? Do you buy it or care to convince me otherwise?

Reply:
What's your take on the above?

Reply:
> Not being familiar with Puppeteer, I find that extremely surprising.

Yeah, I hear you. It's just sort of the nature of true browser testing, unlike the fake browser-like environments some people try to create in Node, e.g. JSDOM. It blew my mind when I first started doing browser testing with Selenium years ago, which has a commonly used equivalent to `page.evaluate()`. One clue that could tell you what's going on is if we use browser APIs in there, like `addEventListener()`. I suppose Deno is blurring that line though since it's adopting many of those APIs, so in the future it may be even harder to tell what's going on

> Anyhow, I'll revert the changes to this test. Do you want me to add a new test file for TimeoutError instead?

No, I think it would be better to update the evaluate callback in this test to return a plain object instead of the `error.toString()` directly. You can then attach properties to that object for whatever you want to assert about outside of the evaluate callback. Puppeteer will serialize that object. IIRC, the reason we aren't doing that already with the `TimeoutError` itself is because Puppeteer has trouble with custom error classes. But a plain object shouldn't be an issue.

> did you see my comments in #300 regarding node-fetch?

Yes. I think you're correct, that it is actually caused by an updated version of Chromium. I forgot that it was happening in the browser tests, which are a special case.

Reply:
Thank you for your guidance. I've reworked the test. See what you think about it.

Reply:
@sholladay Shall we try to close it this weekend?